README.md

```diff
@@ -8,31 +8,43 @@ The UI is bundled with the [Marathon](https://github.com/mesosphere/marathon) pa
 Please note that issues are disabled for this repository. Please feel free to open an issue on the
 [issues page on the main Marathon Repository](https://github.com/mesosphere/marathon/issues?q=is%3Aopen+is%3Aissue+label%3Agui).
 
-#### Setup
+#### Prequisites
+
+Please make sure you've installed and properly configured the following software:
 
-1. Install Mesos and Marathon (follow the [tutorial here](https://mesosphere.github.io/marathon/docs/))
-2. Setup a CORS proxy on your machine to proxy the UI requests to your running Marathon instance (e.g. via [Corsproxy](https://www.npmjs.com/package/corsproxy))
-3. Install [Node 5](https://nodejs.org/en/blog/release/v5.0.0/) and [NPM](https://npmjs.org/)
+* [Docker 1.9](https://www.docker.com/)
+* [Node 5](https://nodejs.org/en/blog/release/v5.0.0/) including [NPM](https://npmjs.org/)
+
```

Comment:
I'd be in favor of supplying both options? Something like, `alternatively you can use the docker-based setup...`?